scipy/spatial/_spherical_voronoi.py

```diff
@@ -166,20 +166,23 @@ class SphericalVoronoi:
     def __init__(self, points, radius=1, center=None, threshold=1e-06):
 
         if radius is None:
-            radius = 1.
+            radius = 1
             warnings.warn('`radius` is `None`. '
                           'This will raise an error in a future version. '
                           'Please provide a floating point number '
                           '(i.e. `radius=1`).',
                           DeprecationWarning)
 
-        self.points = np.array(points).astype(np.double)
-        self.radius = radius
-        self._dim = len(points[0])
+        self.points = np.array(points, dtype=float)
```

Comment:
I'd be surprised if you actually observed a difference switching to `float` from `np.double` on most machines here? Perhaps slightly more correct though, so probably a fair adjustment.

Reply:
I have reverted the change.